docs/source/pre_application/cloud_node_management.rst

```diff
@@ -79,71 +28,64 @@ Now NuCypher Ops is installed we can create a droplet on Digital Ocean:
 
 .. code-block:: bash
 
-    nucypher-ops nodes create --network ibex --count 1 --cloudprovider digitalocean
+    nucypher-ops nodes create
 
 At this point you should see the droplet in your Digital Ocean dashboard.
```

Comment:
Presumably since digital ocean isn't specified in the command - you'll be prompted? Should that be mentioned?

Reply:
✔️